checkov/terraform/checks/resource/ncp/LBTargetGroupUsingHTTPS.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class LBTargetGroupUsingHTTPS(BaseResourceCheck):
+
+    def __init__(self):
+        name = "Ensure LB TargetGroup Using HTTPS"
+        id = "CKV_NCP_15"
+        supported_resources = ("ncloud_lb_target_group",)
+        categories = (CheckCategories.GENERAL_SECURITY,)
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        if "protocol" in conf.keys():
+            if conf.get("protocol")[0] == 'HTTPS':
```

Comment:
```suggestion
            if conf.get("protocol") != ['HTTP']:
```
because `TCP` and `PROXY_TCP` are also fine to use.